.travis.yml

```diff
@@ -28,10 +26,12 @@ before_install:
   - sudo apt-get update
   - sudo apt-get -y install python-pip python-dev libusb-1.0-0-dev
   - pip install awscli --upgrade --user
-  # Install truffle
-  - yarn global add truffle
 script:
-  - bash ./config/travis/build.sh
+  - yarn lint:check
+  - yarn prettier:check
+  - yarn test:coverage
+  - yarn build
+  #- bash ./config/travis/build.sh
```

Comment:
I think we can remove this file, seems that is not used anymore

Reply:
Yes, I left it there for convenience until I confirm that production deployment is successful. Can create a ticket to remember about it

Reply:
Make sense, it's ok for me :)

Reply:
https://app.zenhub.com/workspaces/safe-multisig---web-5ce554debb310a35b2f8b6f8/issues/gnosis/safe-react/1396